ptf/tests/common/base_test.py

```diff
@@ -117,13 +120,10 @@ def get_controller_packet_metadata(p4info, meta_type, name):
                         return m
 
 
-def de_canonicalize_bytes(bitwidth: int, input: bytes):
+def de_canonical(bitwidth: int, input: bytes):
     """
-    This method adds a padding to the 'input' param.
-    Needed for bmv2 since it uses Canonical Bytestrings: this representation
-    trims the data to the lowest amount of bytes needed for that particular value
-    (e.g. 0x0 for PacketIn.ingress_port will be interpreted by Stratum bmv2 using 1 byte, instead of 9 bits,
-    as declared in header.p4)
+    This method adds padding zeros to the 'input' param.
+
     :param bitwidth: the desired size of input.
     :param input: the byte string to be padded.
     :return: padded input with bytes such that: len(bin(input)) >= bitwidth.
```

Comment:
Not related to this PR but doesn't hurt to fix here
```suggestion
    :param bitwidth: the desired size of the output in bits.
    :param input: the byte string to be padded.
    :return: input padded with zero bytes such that: len(bin(input)) = bytewidth.
```